GameData/ROEngines/PartConfigs/RL10A4-2N_BDB.cfg

```diff
@@ -0,0 +1,213 @@
+
+	tags = rl10, rl10a, rl10a-4-2n, 4, rl-10, hydrolox, centaur
+
+	engineType = RL10
+	stagingIcon = LIQUID_ENGINE
+	stageOffset = 1
+	childStageOffset = 1
+
+	TechRequired = hydrolox1992
+	cost = 500
+	entryCost = 26000
+
+	MODULE
+	{
+		name = ModuleEnginesRF
+		thrustVectorTransformName = thrustTransform
+		engineID = RL10A4
+	}
+
+	MODULE
+	{
+		name = ModuleGimbal
+		gimbalTransformName = gimbalTransform
+		gimbalRange = 4
+		useGimbalResponseSpeed = true
+		gimbalResponseSpeed = 16
+	}
+
+	MODULE
+	{
+		name = ModuleSurfaceFX
+		thrustProviderModuleIndex = 1
+		fxMax = 1
+		maxDistance = 120
+		falloff = 2
+		thrustTransformName = thrustTransform
+	}
+
+	MODULE
+	{
+		name = ROLModelConstraint
+		LOOK_CONST
+		{
+			targetName = gimbal1_Lower
+			transformName = gimbal1_Upper
+		}
+		LOOK_CONST
+		{
+			targetName = gimbal1_Upper
+			transformName = gimbal1_Lower
+		}
+		LOOK_CONST
+		{
+			targetName = gimbal2_Lower
+			transformName = gimbal2_Upper
+		}
+		LOOK_CONST
+		{
+			targetName = gimbal2_Upper
+			transformName = gimbal2_Lower
+		}
+	}
+
+	MODULE
+	{
+		name = FXModuleThrottleEffects
+		fxModuleNames = heatColor
+		responseSpeed = 1.0
+		dependOnEngineState = True
+		dependOnThrottle = True
+	}
+
+	MODULE
+	{
+		name = FXModuleThrottleEffects
+		fxModuleNames = throttleColor
+		responseSpeed = 1.0
+		dependOnEngineState = True
+		dependOnThrottle = True
+	}
+
+	MODULE
+	{
+		name = ModuleColorChanger
+		moduleID = heatColor
+		animRate = 0.025
+		shaderProperty = _EmissiveColor
+		excludedRenderer = innerGlow
+		toggleInEditor = false
+		toggleInFlight = false
+		redCurve
+		{
+			key = 1 1 1 1
+			key = 1 1 1 1
+		}
+		greenCurve
+		{
+			key = 0 0 1 1
+			key = 1 1 1 1
+		}
+		blueCurve
+		{
+			key = 0 0 1 1
+			key = 1 1 1 1
+		}
+		alphaCurve
+		{
+			key = 0 0 1 1
+			key = 1 1 1 1
+		}
+	}
+
+	MODULE
+	{
+		name = ModuleColorChanger
+		moduleID = throttleColor
+		animRate = 1
+		shaderProperty = _TintColor
+		includedRenderer = innerGlow
+		toggleInEditor = false
+		toggleInFlight = false
+		redCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+		greenCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+		blueCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+		alphaCurve
+		{
+			key = 0 0
+			key = 1 1
+		}
+	}
+}
+
+@PART[ROE-RL10A4-2N]:AFTER[RealismOverhaulEngines]
+{
+	@title = RL10A-4-2N Series Vacuum Engine
+	@MODULE[ModuleEngineConfigs] 
+	{
+		@configuration = RL10A-4-1-2
+		@origMass = 0.170
+		@CONFIG
+		{
+			@name = RL10A-4-1-2
+			@atmosphereCurve
+			{
+				@key = 0 451.0
+			}
+		}
```

Comment:
Which CONFIG entry are you editing?  There should be an RL10A-4-1-2 added in RO, and it should get the correct atmosphere curve, and this patch should just delete unwanted engine configs.  I guess this is sort of okay, but it's weird to use the engine config system and then edit it.  Unless there is a specific reason this config should not be in RO.